share/functions/__fish_complete_gpg_key_id.fish

```diff
@@ -5,13 +5,15 @@ function __fish_complete_gpg_key_id -d 'Complete using gpg key ids' -a __fish_co
     set -l keyid
     $__fish_complete_gpg_command --list-keys --with-colons | while read garbage
         switch $garbage
-            # Extract user ids
+            # Extract user ids (note: gpg escapes colons as '\x3a')
             case "uid*"
-                echo $garbage | cut -d ":" -f 10 | sed -e "s/\\\x3a/:/g" | read uid
+                set -l __uid (echo $garbage | string split ":")
```

Comment:
String doesn't need to read from stdin. Use `string split : -- $garbage`.

Reply:
Oopsie. Cheers!